src/opnsense/scripts/shell/setaddr.php

```diff
@@ -542,6 +542,10 @@ function console_configure_dhcpd($version = 4)
         unset($config['system']['webgui']['noantilockout']);
         $restart_webgui = true;
     }
+    if (empty($config['interfaces'][$interface]['antilockout'])) {
```

Comment:
eh, huh, why? you can use it to configure a WAN too...

Reply:
I assumed the user would not choose "yes" here when configuring a WAN interface, but yes, the user could. It will configure antilockout for the LAN interface now and if no LAN is present, print a warning about it.